inventory/environ.py

```diff
@@ -377,6 +377,24 @@ def getSplunkApps(vars_scope):
                 appList.append(app)
     vars_scope["splunk"]["apps_location"] = appList
 
+    # Check if theres a local apps list for CM/Deployer roles
```

Comment:
Maybe this section should be a separate function? That way we can add [test coverage](https://github.com/splunk/splunk-ansible/blob/develop/tests/small/test_environ.py) for it?

Reply:
Yea, I'll move this to a separate function.